biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/PomConfiguration.java

```diff
@@ -87,4 +87,10 @@
 	// default: 5 Minutes
 	int poll_time(int pollTimeInSecs);
 
+	/**
+	 * Also considers the dependency management section of a pom. Default is
+	 * false.
+	 */
+	boolean depManagement(boolean deflt);
```

Comment:
Can we spell this out to `dependencyManagement`? Then it will match the variable names.